.github/workflows/lint-docs-weekly.yaml

```diff
@@ -0,0 +1,21 @@
+name: Check Markdown links
+
+on:
+  schedule:
+    - cron: "0 9 * * 1"
+   paths:
```

Comment:
I don't think we need branches here, but I'm not sure if this will actually scan every branch on the repo (that could be a good thing to pick up broken links in older versions of the book)